Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -28,6 +28,7 @@ open class MapView: UIView {
 
     /// The `camera` object manages a camera's view lifecycle.
     public private(set) var camera: CameraAnimationsManager!
+    private var cameraAnimatorsRunner: CameraAnimatorsRunnerProtocol!
```

Comment:
I don't mind the reference here. I prefer it to the 2 layers of passthroughs that would be needed otherwise (the public wrapper and the internal impl).